docs/configuration.rst

```diff
@@ -897,8 +897,9 @@ Each acl contains a list of rules: a packet will have the first matching rule
 applied to it.
 
 Each rule is a dictionary containing the single key 'rule' with matches
-and actions. Matches are key/values based on the ryu RESTFul API. Actions
-is a dictionary of actions to apply upon match.
+and actions. Matches are key/values based on the ryu RESTFul API.
+<https://ryu.readthedocs.io/en/latest/app/ofctl_rest.html#reference-description-of-match-and-actions>
```

Comment:
Syntax is still a little wrong here, you want to use the following exactly (RST is a bit weird for URLs):

```
`ryu RESTFul API
<https://ryu.readthedocs.io/en/latest/app/ofctl_rest.html#reference-description-of-match-and-actions>`_

Check out the embeddable URI documentation here: https://docutils.sourceforge.io/docs/user/rst/quickref.html#external-hyperlink-targets
```

Reply:
My bad, updated this and did a manual inspection with the python previewer.  This should be corrected in the new patch.